dll/win32/shell32/CShellLink.cpp

```diff
@@ -2802,6 +2802,29 @@ BOOL CShellLink::OnInitDialog(HWND hwndDlg, HWND hwndFocus, LPARAM lParam)
     if (m_sDescription)
         SetDlgItemTextW(hwndDlg, IDC_SHORTCUT_COMMENT_EDIT, m_sDescription);
 
+    /* Hot key */
+    SendDlgItemMessageW(hwndDlg, IDC_SHORTCUT_KEY_HOTKEY, HKM_SETHOTKEY, m_Header.wHotKey, 0);
+
+    /* Run */
+    const WORD runstrings[] = { IDS_SHORTCUT_RUN_NORMAL, IDS_SHORTCUT_RUN_MIN, IDS_SHORTCUT_RUN_MAX };
+    const BYTE runshowcmd[] = { SW_SHOWNORMAL, SW_SHOWMINNOACTIVE, SW_SHOWMAXIMIZED };
+    HWND hRun = GetDlgItem(hwndDlg, IDC_SHORTCUT_RUN_COMBO);
+    for (UINT i = 0; i < _countof(runstrings); ++i)
+    {
+        WCHAR buf[MAX_PATH];
+        if (!LoadString(shell32_hInstance, runstrings[i], buf, _countof(buf)))
+            break;
+
+        int idx = SendMessage(hRun, CB_ADDSTRING, 0, (LPARAM)buf);
+        if (idx < 0)
+            continue;
+        SendMessage(hRun, CB_SETITEMDATA, idx, runshowcmd[i]);
```

Comment:
```suggestion
        SendMessageW(hRun, CB_SETITEMDATA, idx, runshowcmd[i]);
```

Reply:
Why? There is no string conversion here.

Reply:
For consistency. Why would you here rely on the SendMessage ANSI/UNICODE macro and not just call directly the actual underlying function? (since everywhere else the explicit unicode function is always used)